.coderabbit.yaml

```
@@ -0,0 +1,141 @@
+    - path: "**/packages/*/src/*.test.tsx"
+      instructions: |
+        Review test files for:
+        - English test descriptions (convert Korean descriptions to English)
+        - Comprehensive test coverage including visual, behavioral, and accessibility tests
+        - Theme compatibility testing - ensure components work with different theme variants
+        - Proper use of @testing-library/react patterns
+        - Testing component variants, props, and edge cases
+        - ThemeProvider wrapper in tests when testing theme-dependent components
+        - Consistent test structure and naming conventions
+        - Proper setup and cleanup in test files
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)